docs/pages/foundations/animation/implementation.js

```diff
@@ -0,0 +1,47 @@
+// @flow strict
+import { type Node } from 'react';
+import { Flex } from 'gestalt';
+import Page from '../../../docs-components/Page.js';
+import PageHeader from '../../../docs-components/PageHeader.js';
+import MainSection from '../../../docs-components/MainSection.js';
+import SandpackExample from '../../../docs-components/SandpackExample.js';
+import useExample from '../../../examples/animation/useExample.js';
+
+export default function AnimationImplementation(): Node {
+  return (
+    <Page title="Implementing animations">
+      <PageHeader
+        badge="pilot"
+        name="Implementing animations"
+        type="guidelines"
+        description="There are multiple ways to implement animations in product."
+      />
+      <MainSection
+        name="Gestalt"
+        description="Many of our components come with animations baked in! If you are implementing a custom animation, be sure to wrap the animation in the [useReducedMotion hook](https://gestalt.pinterest.systems/web/utilities/usereducedmotion) to ensure an accessible experience. This hook will automatically disable any contained animations when the user enables the Reduced Motion setting on their device."
```

Comment:
This is very web-specific. Perhaps this section should be "Gestalt on web"?

Also: `useReducedMotion` doesn't wrap anything, or do anything automatically. It returns a boolean that must be used by the developer in their logic to show/hide the animation.